### ISSUE TITLE
strange slide number under standout frame

### ISSUE THREAD

**Issue by leomao** [matze/mtheme#341]
I am using Archlinux with up-to-date texlive packages.  After update texlive recently, I encounter strange slide number for standout frame. With the following minimal example:
```tex
\documentclass{beamer}
\usetheme{metropolis}           % Use metropolis theme
\title{A minimal example}
\date{\today}
\author{Matthias Vogelgesang}
\institute{Centre for Modern Beamer Themes}
\begin{document}
  \maketitle
  \section{First Section}
  \begin{frame}{First Frame}
    Hello, world!
  \end{frame}
  \begin{frame}[standout]
    Thanks!
  \end{frame}
\end{document}
```
The last frame will have non-inverted background and slide number at the bottom:
![output](https://user-images.githubusercontent.com/1811303/45208016-263e1a00-b2bc-11e8-80a9-5231c9fb3915.png)

I think recent updates of texlive cause the problem. Maybe some changes in recent beamer packages break the compatibility?

**Comment by phiadaarr** [comment]
I observe the same behaviour.

**Comment by louisstuart96** [comment]
A recent change in `beamercolorbox` may cause this problem, which is fixed in beamer v3.53. 
If you're using TeX Live from Arch repository, the update may not be pushed in time. 

You can directly download `beamerbasecolor.sty` from CTAN package or `beamer` GitHub repository to override current one.

**Comment by beojan** [comment]
You can also try adding the `plain` option to your frame.
